test/unit/nfts/seasons/SeasonBase.t.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity 0.8.18;
+
+import {BaseTest} from "@test/BaseTest.sol";
+
+// TODO is this needed or over kill?
```

Comment:
I'm in two minds about this - might be overkill, but might be worth having the appropriate coverage anyway.

Reply:
@utx0 - we can move this into another PR also.